src/components/DatePicker.tsx

```diff
@@ -11,6 +10,8 @@ import Button from "./Button";
 import TextField, { Props as TextFieldProps } from "./TextField";
 import Touchable from "./Touchable";
 import theme from "../styles/DefaultTheme";
+//@ts-ignore
```

Comment:
any reason you need this here? is it because of the .native and .web extension? i think this will solve your issue:

https://stackoverflow.com/questions/44001050/platform-specific-import-component-in-react-native-with-typescript